src/marine/smap_sss2ioda.py

```diff
@@ -17,7 +16,7 @@
 import dateutil.parser
 from pathlib import Path
 
-IODA_CONV_PATH = Path(__file__).parent/"@SCRIPT_LIB_PATH@"
+IODA_CONV_PATH = Path(__file__).parent/"../lib/pyiodaconv"
```

Comment:
The other scripts have the deleted version of this line (ie the one using `@SCRIPT_LIB_PATH@`). Shouldn't this be the case here as well?

Reply:
in smos_sss2ioda.py, the 
IODA_CONV_PATH = Path(__file__).parent/"@SCRIPT_LIB_PATH@"
does not exist, so this converter too does not have it.
Let me know.

Reply:
I think the `@SCRIPT_LIB_PATH@` version is needed to get the `make install` version of the script to work correctly. However, I'm not certain. @aerorahul or @CoryMartin-NOAA could you confirm this? Thanks!

Reply:
Yes that is my understanding, because it may not be stored in ../lib/pyiodaconv

Reply:
@srherbener, I will add that. However, I will have to add to the smos_sss2ioda.py as well.

Reply:
@ShastriPaturi that sounds great, and thanks for taking care of this as well as the smos script.